distro/webjar/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.camunda.bpm</groupId>
+    <artifactId>camunda-parent</artifactId>
+    <relativePath>../../parent</relativePath>
+    <version>7.5.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.camunda.bpm.webapp</groupId>
+  <artifactId>camunda-webapp-webjar</artifactId>
+  <packaging>jar</packaging>
+  <name>camunda BPM - webapp - webjar</name>
+
+  <properties>
+    <skipTests>true</skipTests>
+  </properties>
+
+  <!-- add dependency so this is run after the war was build -->
+  <dependencies>
+    <dependency>
+      <groupId>org.camunda.bpm.webapp</groupId>
+      <artifactId>camunda-webapp</artifactId>
+      <version>${project.parent.version}</version>
+      <type>jar</type>
```

Comment:
I think this dependency should point to the war file. Although both artifacts are probably built by the same Maven project.